thewalrus/tests/test_hafnian_approx.py

```diff
@@ -51,6 +51,6 @@ def test_approx_negative_error():
 def test_ones_approx(n):
     """Check hafnian_approx(J_2n)=(2n)!/(n!2^n)"""
     A = np.float64(np.ones([2 * n, 2 * n]))
-    haf = hafnian(A, approx=True, num_samples=1e4)
+    haf = hafnian(A, approx=True, num_samples=10000)
```

Comment:
Is the numba version less accurate?

Reply:
this is because numba does not like to get a float when it expects an integer.

Reply:
oh... yeah of course. I think I counted the number of zeros incorrectly when I left that comment 😬